packages/ui-expandable/src/Expandable/index.tsx

```diff
@@ -43,9 +43,7 @@ class Expandable extends Component<ExpandableProps, ExpandableState> {
   static propTypes = propTypes
   static allowedProps = allowedProps
   static defaultProps = {
-    defaultExpanded: false,
-    onToggle: function () {},
-    children: null
```

Comment:
this is just removing some old old code, ancient InstUI loved default values even when it was not needed.